### PR TITLE
Update The Open Source Page to include Assemble and The New Community Newsletter

### PR DESCRIPTION
**Pull request by radioblahaj**
I was looking at hackclub.com/opensource and noticed that Assemble and the newsletter weren't there. So I made a quick PR to add them.

### CODE REVIEW

File: pages/opensource.js [hunk 1]
```diff
@@ -150,6 +158,10 @@ const Page = ({ repos }) => (
         <BankProject
           name="VIP Newsletters"
           url={`https://github.com/hackclub/vip-newsletters`}
+        />
+         <BankProject
+          name="Community Newsletters"
+          url="https://hackclub.com/newsletters
```
Comment (sampoder) [inline pages/opensource.js]:
```suggestion
          url="https://hackclub.com/newsletters"
```

File: pages/opensource.js [hunk 2]
```diff
@@ -150,6 +158,10 @@ const Page = ({ repos }) => (
         <BankProject
           name="VIP Newsletters"
           url={`https://github.com/hackclub/vip-newsletters`}
+        />
+         <BankProject
+          name="Community Newsletters"
+          url="https://hackclub.com/newsletters"
```
Comment (sampoder) [inline pages/opensource.js]:
```suggestion
          url="https://hackclub.com/newsletter"
```

File: pages/opensource.js [hunk 3]
```diff
@@ -150,6 +158,10 @@ const Page = ({ repos }) => (
         <BankProject
           name="VIP Newsletters"
           url={`https://github.com/hackclub/vip-newsletters`}
+        />
+         <BankProject
+          name="Community Newsletters"
```
Comment (sampoder) [inline pages/opensource.js]:
```suggestion
          name="Community Newsletter"
```

File: pages/opensource.js [hunk 4]
```diff
@@ -150,6 +158,10 @@ const Page = ({ repos }) => (
         <BankProject
           name="VIP Newsletters"
```
Comment (sampoder) [inline pages/opensource.js]:
```suggestion
          name="VIP Newsletter"
```

**Review by sampoder — APPROVED** [review]
Hack yeah! Thank you so much for helping with this

**Comment by radioblahaj** [comment]
:yay:

**Comment by radioblahaj** [comment]
Of course! Glad I could help in some way.

**Pull request merged**